Cython/Utility/ObjectHandling.c

```diff
@@ -1790,20 +1820,35 @@ static PyObject *__Pyx_SelflessCall(PyObject *method, PyObject *args, PyObject *
     Py_DECREF(selfless_args);
     return result;
 }
+#else
+static PyObject *__Pyx_SelflessCall(PyObject *method, PyObject *const *args, Py_ssize_t nargs, PyObject *kwnames) {
+    return
+#if PY_VERSION_HEX < 0x03090000
+    _PyObject_Vectorcall
+#else
+    PyObject_Vectorcall
+#endif
+        (method, args ? args+1 : NULL, nargs ? nargs-1 : 0, kwnames);
+}
+#endif
```

Comment:
Can `args` really be `NULL`?

Reply:
https://docs.python.org/3/c-api/call.html#the-vectorcall-protocol

> `args` is a C array consisting of the positional arguments followed by the values of the keyword arguments. This can be `NULL` if there are no arguments.